examples/radial_gradient_filters.py

```diff
@@ -0,0 +1,57 @@
+"""
+=======================
+Radial Gradient Filters
+=======================
+
+This example applies both the normalizing radial gradient (`sunkit_image.radial.nrgf`) filter and Fourier
+normalizing radial gradient filter (`sunkit_image.radial.fnrgf`) to a sunpy map.
+
+
+"""
+import numpy as np
+import matplotlib.pyplot as plt
+
+import astropy.units as u
+
+import sunpy.map
+import sunpy.data.sample
+
+import sunkit_image.radial as radial
+from sunkit_image.utils import equally_spaced_bins
+
+###########################################################################
+
```

Comment:
```suggestion
```

Reply:
If I remove this then the comments will appear outside the code block in the generated html, is that alright?

Reply:
Yes.